video/out/wayland_common.c

```diff
@@ -1552,10 +1564,38 @@ static void apply_keepaspect(struct vo_wayland_state *wl, int *width, int *heigh
     if (!wl->opts->keepaspect)
         return;
 
+    int phys_width = handle_round(wl->scaling, *width);
+    int phys_height = handle_round(wl->scaling, *height);
+
+    // Ignore bogus initial events so we don't choose the wrong direction.
```

Comment:
What are these bogus initial events? Removing this doesn't cause any issues on sway master

Reply:
Ah I see, makes sense to me then. LGTM with this comment reworded

Reply:
Reworded.